clang/test/CodeGen/asm-inout.c

```diff
@@ -38,11 +38,11 @@ int test4(volatile int *addr) {
   return (int)oldval;
 }
 
-// This should have both inputs be of type x86_mmx.
+// This should have both inputs be of type <1 x i64>.
```

Comment:
No conflict, although the comment doesn't really have any value now. Since Clang's `__m64` is naturally just <1 x i64>, that's not particularly worthy of comment.

Note below there's a case of a <8 x i8> vector passed to a 'y' constraint.